src/components/orderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,299 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+// import {
```

Comment:
Remove commented lines if not needed

Reply:
Commented these lines since we are using hardcoded JSON for now. Will uncomment them once we use actual APIs.